code/src/pages/cart/CartPage.tsx

```diff
@@ -19,118 +22,113 @@ type CartItemsType = {
-  );
-
-  const handleQuantityChangeWrapper =
-    (index: number) => (newQuantity: number) => {
-      handleQuantityChange(index, newQuantity);
-    };
-
-  const handleOnDeleteWrapper = (index: number) => () => {
-    handleDeleteItem(index);
-  };
-
-  const cartItemsBlock = cartItems.map((item, index) => (
-    <CartItem
-      key={item.name}
-      item={item}
-      onQuantityChange={handleQuantityChangeWrapper(index)}
-      onDelete={handleOnDeleteWrapper(index)}
-    />
+  const cartItemsBlock = cartItems?.items.map((item: CartItem) => (
+    <CartItem key={item.productId} item={item} />
   ));
 
+  const totalPrice = formatPrice(cartItems?.totalPrice ?? 0);
+
   return (
-    <AppBox className="spa-cart-page">
-      <AppBox className="spa-cart-page__content">
-        <AppBox className="spa-cart-page__items">
-          <AppTypography
-            className="spa-cart-page__items--label"
-            variant="h3"
-            component="h1"
-            translationKey="myCart.label"
-          />
-          {cartItemsBlock}
-        </AppBox>
-        <AppBox className="spa-cart-page__order-summary">
-          <AppTypography
-            className="spa-cart-page__order-summary--label"
-            variant="h1"
-            component="h2"
-            translationKey="orderSummary.label"
-          />
-          <AppBox className="spa-order-summary__details">
-            <AppBox className="spa-order-summary__row">
-              <AppTypography
-                component="p"
-                className="spa-order-summary__text"
-                translationKey="subtotal.label"
-              />
-              <AppTypography
-                component="p"
-                className="spa-order-summary__text"
-                variant="subtitle2"
-              >
-                {formatPrice(subtotalItems)}
-              </AppTypography>
-            </AppBox>
-            <AppBox className="spa-order-summary__row">
-              <AppTypography component="p" translationKey="delivery.label" />
-              <AppTypography component="p" translationKey="free.label" />
-            </AppBox>
+    <PageWrapper>
+      <AppBox className="spa-cart-page">
+        <AppBox className="spa-cart-page__content">
+          <AppBox className="spa-cart-page__items">
             <AppTypography
-              component="p"
-              className="spa-order-summary__underline-text"
-              translationKey="country.label"
+              className="spa-cart-page__items--label"
+              variant="h3"
+              component="h1"
+              translationKey="myCart.label"
+              data-testid="myCartLabel"
             />
-            <AppBox className="spa-order-summary__row spa-order-summary__total-line">
+            {cartItemsBlock}
+          </AppBox>
+          <AppBox className="spa-cart-page__order-summary">
+            <AppTypography
+              className="spa-cart-page__order-summary--label"
+              variant="h1"
```

Comment:
Smth strainge with variant and component
If we have variant h2 for what reasone component h2?